pkg/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -125,14 +125,16 @@ type MachineConfig struct {
 	MachineInstall *InstallConfig `yaml:"install,omitempty"`
 	//   description: |
 	//     Allows the addition of user specified files.
+	//     If an `op` value of `append` is used, the existing file will be appended.
```

Comment:
I guess we need to document that `op` can be one of: `write`, `append`, as `write` is not in the docs?

Reply:
Done